source/developer/localization.rst

```diff
@@ -15,22 +15,25 @@ To translate a language:
 
 1 - Join the `Mattermost Translation Server <http://translate.mattermost.com>`__.
 
-2 - Confirm that the language you want to help translate is listed in the `Translation Server <http://translate.mattermost.com>`__.
+2 - Confirm that the language you want to help translate is listed in the shipped project of the `Translation Server <https://translate.mattermost.com/projects/mattermost/mattermost-server_master/>`__.
+If it is, perhaps offer to help translate or review and make suggestions.
 
-    - If it is, perhaps offer to help translate or review and make suggestions.
-    - If not, ask to setup your language on the `Mattermost localization channel <https://community.mattermost.com/core/channels/localization>`__, so you can start helping with the translation.
+3 - Confirm that the language you want to help translate is listed in the work in progress project of the `Translation Server <https://translate.mattermost.com/projects/i18n-wip/mattermost-server-wip/>`__.
+If it is, perhaps offer to help translate or review and make suggestions.
 
-3 - Review translation rules written by localization leads, when applicable:
+4 - The language you want to help translate is neither in the shipped project nor in the work in progress `projects <https://translate.mattermost.com/projects/>`__.
+
+5 - Review translation rules written by localization leads, when applicable:
 
     - `French (Règles pour la traduction francophone de Mattermost) <https://github.com/wget/mattermost-localization-french-translation-rules>`__
     - `German (Regeln zur Übersetzung von Mattermost) <https://gist.github.com/meilon/1317a9425988b3ab296c894a72270787>`__
 
 Translations Updates
 --------------------
 
-1 - New and updated strings are imported to the `Mattermost Translation Server <http://translate.mattermost.com>`__ each day at 00:00 GMT. Strings with a ``mobile`` prefix are imported from the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__ project, others are from `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__.
+1 - New and updated source strings are being automatically pulled on GitHub pushes by developers.
 
-2 - A new pull request with latest translations that reach at least Beta Quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
+2 - A new pull request with latest translations that reach at least Beta quality is submitted to the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__, `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__ repos each Monday at 22:00 GMT.
```

Comment:
Do we still have PRs submitted once per week on Mondays?

Reply:
With some new changes I did, it should.

Reply:
Awesome!